CHANGELOG.md

```diff
@@ -99,6 +99,7 @@
 ### Documentation
 
 * [CHANGE] Clarify what deprecation means in the lifecycle of configuration parameters. #4499
+* [CHANGE] Update compactor split-group/split-and-merge-shards recommendation on component page. #4623
```

Comment:
The `/` is often ambiguous. What word can replace it?

Reply:
Changed!